runners/wikiConsole/templates/npmInstall.asciidoc

```diff
@@ -0,0 +1,19 @@
+npmInstall
```

Comment:
The header should not only be the name of the command. It should describe shortly what is done